core/crypto/eddsa/eddsa.go

```diff
@@ -144,13 +145,9 @@ func (k *PublicKey) ToPEMFile(f string) error {
 
 // ToECDH converts the PublicKey to the corresponding ecdh.PublicKey.
 func (k *PublicKey) ToECDH() *ecdh.PublicKey {
-	var dhBytes, dsaBytes [32]byte
-	copy(dsaBytes[:], k.Bytes())
-	defer utils.ExplicitBzero(dsaBytes[:])
-	extra25519.PublicKeyToCurve25519(&dhBytes, &dsaBytes)
-	defer utils.ExplicitBzero(dhBytes[:])
+	ed_pub, _ := new(edwards25519.Point).SetBytes(k.Bytes())
```

Comment:
is there an equivalent to ExplicitBzero for edwards25519.Point ? I looked at the available methods but did not see a way to clear this before letting it be garbage collected

Reply:
i'm guessing we'd have to fork the edswards25519 module to get it to wipe the bytes... i'm fine if you want to do that on a second pass. or maybe ask for the feature addition or submit a pull/merge request.

Reply:
marking resolved, we can discuss this in https://github.com/katzenpost/katzenpost/pull/25#discussion_r885850820 or in a separate issue thread